caddyhttp/rewrite/to.go

```diff
@@ -28,6 +28,10 @@ func To(fs http.FileSystem, r *http.Request, to string, replacer httpserver.Repl
 			query = tparts[1]
 		}
 
+		if strings.HasSuffix(tparts[0], "/") && !strings.HasSuffix(t, "/") {
+			t += "/"
+		}
+
```

Comment:
With this, I do not think the similar check below is needed anymore. Maybe move the `// add trailing slash for directories, if present` comments over here and delete the other one.